packages/ibm-products/src/components/ConditionBuilder/utils/useTranslations.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright IBM Corp. 2021, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useContext } from 'react';
+import { ConditionBuilderContext } from '../ConditionBuilderContext/ConditionBuilderProvider';
+import { translationsObject } from '../ConditionBuilderContext/translationObject';
+
+export const useTranslations = (translationKeys) => {
+  const { translateWithId } = useContext(ConditionBuilderContext);
+
+  return translationKeys.map((translationKey) =>
```

Comment:
i'm personally not a fan of this conditional notation. i think readability is improved by using the standard `if (...) return` notation. also i could include a `console.warn` if they key isn't found in the translation object to improve debugging.

Reply:
modified, 
but console.warn may not required here , because if the key is not found, we will return key itself.